tests/E2ETests/test_pytom_FullPipeline.py

```diff
@@ -466,22 +439,18 @@ def test_14_UpdateParticleList(self):
         pl = ParticleList()
         pl.fromXMLFile(self.plFilenameReduced)
 
-        # pl2 = ParticleList()
-        #
-        # for p in pl:
-        #     if p.getScore().getValue() > 0.09:
-        #         pl2.append(p)
-        # print('\n\n\n\n', len(pl2), '\n\n\n')
-        # pl2.toXMLFile(self.plFilenameReduced)
-
         self.assertTrue(pl[0].getWedge().getWedgeAngle() == 70, 'Wedge angles not updated')
 
     def test_15_CTFCorrection(self):
         """
         check that resulting scores of alignment of corrected are similar to ref values
         """
-        if which('ctfphaseflip') is None: raise self.skipTest('No ctfphaseflip from imod installed')
-        import glob
+        if shutil.which('ctfphaseflip') is None:
+            raise self.skipTest('\n>>>>>>>>>>>>>>>> No ctfphaseflip from imod installed, meaning the test will be '
+                                'completed without the strip-based phase flipping from IMOD\n')
+        else:
+            self.ctf_correction = True
+
```

Comment:
I would like to see the assigning of `self.ctf_correction=True` inside the setup and change this `if` statement to `if not self.ctf_correction`

(this is to prevent strange behaviour when I try to run `--failed-first` or just a single test function

Reply:
Done!